cli/pull.go

```diff
@@ -10,6 +11,7 @@ import (
 
 	"github.com/alibaba/pouch/ctrd"
 	"github.com/alibaba/pouch/pkg/reference"
+	"golang.org/x/crypto/ssh/terminal"
```

Comment:
Add a blank line before "golang.org/x/crypto/ssh/terminal", because 

> When importing packages, to improve readabilities, we should import package by sequence: system packages, project's own packages and third-party packages. And we should keep a blank line among these three kinds of packages;

in https://github.com/alibaba/pouch/blob/master/docs/contributions/code_styles.md

Reply:
Got it.  Have Updated